config/notebook_config.py

```diff
@@ -0,0 +1,56 @@
+'''
+    Import the needed functionality
+    - tkinter : 
+        Python GUI library
+    - configuration.ProjectConfiguration
+        Configuration object that reads/writes to the configuration settings YAML file.
+    - configurationui.SettingsUpdate
+        tkinter based UI that dynamically loads any appropriate configuration file
+        and displays it to the user to alter the settings.
+'''
+from tkinter import * 
+from configuration import ProjectConfiguration
+from configurationui import SettingsUpdate
+
+project_configuration_file = "./project.yml"
+
+def configure_settings(configuration_yaml = None):
+    '''
+        ProjectConfiguration will open an existing YAML file or create a new one. It is
+        suggested that your project simply create a simple configuration file containing
+        all of you settings so that the user simply need to modify it with the UI. 
+
+        In this instance, we assume that the default configuration file is called project.yml.
+        This will be used if the user passes nothing else in. 
+    '''
```

Comment:
I also like to include the parameter and return comments in the docstring. This text pops up in IDE when using method. 

Example: 
    :param datastore_rg: Resource Group containing Azure Storage Account
    :return: Pointer to Azure Machine Learning Blob Datastore

Reply:
Typically I do as well, but for this one I updted the doc string to explain it better.